oops/backplane/distance.py

```diff
@@ -140,3 +140,292 @@ def center_time(self, event_key):
 Backplane._define_backplane_names(globals().copy())
 
 ################################################################################
+
+
+
+
+################################################################################
+# UNIT TESTS
+################################################################################
+
+import unittest
+from oops.meshgrid                      import Meshgrid
+from oops.unittester_support import     TESTDATA_PARENT_DIRECTORY
+from oops.constants                     import DPR
+from oops.backplane.unittester_support  import show_info
+
+
+#===========================================================================
+def exercise_observer(bp, printing, saving, dir, refdir,
+                      planet=None, moon=None, ring=None,
+                      undersample=16, use_inventory=False, inventory_border=2):
+    """generic unit tests for distance.py"""
+
+    if planet != None:
+        test = bp.distance(planet)
+        show_info(bp, 'Distance observer to planet (km)', test,
+                  printing=printing, saving=saving, dir=dir, refdir=refdir)
+
+        test = bp.distance(planet, direction='dep')
+        show_info(bp, 'Distance observer to planet via dep (km)', test,
+                  printing=printing, saving=saving, dir=dir, refdir=refdir)
+
+        test = bp.center_distance(planet)
+        show_info(bp, 'Distance observer to planet center (km)', test,
+                printing=printing, saving=saving, dir=dir, refdir=refdir)
+
+        test = bp.distance(planet+':limb')
+        show_info(bp, 'Distance observer to planet limb (km)', test,
+                  printing=printing, saving=saving, dir=dir, refdir=refdir)
+
+        ### TODO: use Body.has_rings() once Mark has created it
+        try:
+            test = bp.distance(planet+':ansa')
+            show_info(bp, 'Distance observer to ansa (km)', test,
+                      printing=printing, saving=saving, dir=dir, refdir=refdir)
+        except:
```

Comment:
In general we don't want a naked `except`. It would be nice to know what you're looking for here.

Reply:
Not resolved. If the tests don't work, perhaps just comment them out? There are similar examples in several other backplane files.